packages/zowe-explorer/src/extension.ts

```diff
@@ -39,14 +38,14 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
 
     await ProfilesUtils.initializeZoweProfiles((msg) => ZoweExplorerExtender.showZoweConfigError(msg));
     await Profiles.createInstance(ZoweLogger.imperativeLogger);
-    initializeSpoolProvider(context);
+    await Profiles.getInstance().getProfileInfo();
```

Comment:
Can this line be moved inside `Profiles.createInstance`? It seems good to minimize the amount of code needed in `extension.ts` if possible 😋

Reply:
Sure, good call -- addressed in next commit